src/components/dialogs/export-dialog.jsx

```diff
@@ -79,6 +82,23 @@ const ExportDialog = ({
             });
     }, [studyUuid, snackError]);
 
+    const fetchNodeName = useCallback(
```

Comment:
You don't need to do a fetch for node name,  it is stored in redux "CurrentTreeNodeData"
you can get it from the redux like CurrentTreeNode.data.label

Reply:
nodeUuid is passed by contextual menu, not the current node selected, so we can not use currentTreeNode in redux
In contrast, we can use another item in redux, i.e.   

```js
    const treeModel = useSelector(
        (state) => state.networkModificationTreeModel
    );
    const	nodeName = treeModel.find((node) => node.id === nodeUuid))?.data.label
```

Reply:
Thanks.